config/locales/views/enrichment/show.en.yml

```diff
@@ -22,5 +22,11 @@ en:
           - providing an opportunity for students to pursue their learning in their own area of interest and strength
           - studying concepts to a greater depth and complexity
           - encouraging your class to pursue wider educational and further career goals
-          - supporting you to achieve the %{secondary_certificate}
+          - |
+            supporting you to achieve the %{secondary_certificates}
```

Comment:
This line should say - supporting you to achieve the **following** secondary certificates